sieve_of_Eratosthenes.py

```diff
@@ -0,0 +1,22 @@
+def sieve_of_eratosthenes(n):
+    
+    prime = [True for i in range(n+1)]
+    p = 2
+    while (p * p <= n):
+        # If prime[p] is not changed, then it is a prime
+        if (prime[p] == True):
+            # Update all multiples of p
+            for i in range(p * 2, n+1, p):
+                prime[i] = False
+        p += 1
+    # Print all prime numbers
+    for p in range(2, n):
+        if prime[p]:
+            print(p)
+
+
+if __name__=='__main__':
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
+++ b/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
@@ -15,7 +15,7 @@
             print(p)
 
 
-if __name__=='__main__':
+if __name__ == '__main__':
     n = 30
     print("Following are the prime numbers smaller")
     print("than or equal to", n)
```